modules/examples/include/hephaestus/examples/types_protobuf/accumulator.h

```diff
@@ -0,0 +1,29 @@
+//=================================================================================================
+// Copyright (C) 2023-2024 HEPHAESTUS Contributors
+//=================================================================================================
+
+#pragma once
+
+#include "hephaestus/examples/types/accumulator.h"
+#include "hephaestus/examples/types/proto/accumulator.pb.h"
+#include "hephaestus/serdes/protobuf/concepts.h"
+
+namespace heph::serdes::protobuf {
```

Comment:
that would be awesome, how would you do it? like `reflect-cpp` can do it for flatbuffer

Reply:
macro? :monkey_face:

Reply:
that could be an idea, I am not against that, but you need to run the macro and save the file to git so that other module can have access to the file